src/background/deinflect.test.ts

```diff
@@ -318,6 +318,31 @@ describe('deinflect', () => {
     expect(match!.reasonChains).not.toContainEqual([Reason.Continuous]);
   });
 
+  it('deinflects respectful continuous forms', () => {
+    // prettier-ignore
+    const cases: Array<[string, string, Array<Array<Reason>>]> = [
+      ['行っていらっしゃる', '行く', [[Reason.Respectful, Reason.Continuous]]],
+      ['行っていらっしゃい', '行く', [[Reason.Respectful, Reason.Continuous, Reason.Imperative], [Reason.MasuStem]]],
```

Comment:
@birtles We should probably remove the masu-stem reason in these cases as well, right?

Reply:
If it's not too difficult, I think that would be ideal.